### PR TITLE
Redesign Introduction section for the annex

### PR DESCRIPTION
**Pull request by DanilChapovalov**
Introduction reworked to align with the current state of the specification:
More attention is given to the key concepts of RTP Frame and Decode target,
added short description of the purpose of Refered frames, DTIs and Chains.
Detailed introduction to the Templates is moved to own section leaving behind only a short paragraph.

This partially address issue#135

### CODE REVIEW

**Comment by aboba** [comment]
The term SFU is included in this PR instead of SFM.

File: av1-rtp-spec.md [hunk 1]
```diff
@@ -664,14 +664,18 @@ End-to-end security services such as authentication, integrity, or confidentiali
...
+The DD uses the concept of Decode targets, each of which represents a subset of a scalable video stream necessary to decode the stream at a certain temporal and spatial fidelity. A frame may be associated with several Decode targets. This concept is used to facilitate selective forwarding, as is done by a Selective Forwarding Unit (SFU). Typically an SFM would select one Decode target for each endpoint, and forward all frames associated with that target.
```
Comment (Philipel-WebRTC) [inline av1-rtp-spec.md:671]: "... by a Selective Forwarding Unit (SFU). Typically an SFM ..."

File: av1-rtp-spec.md [hunk 2]
```diff
@@ -1114,6 +1118,11 @@ Chains protecting no active decode targets MUST be ignored.
 **Note:** To increase the chance of using a predefined template, chains protecting no active decode targets may refer to any frame, including an RTP frame that was never produced.
 {:.alert .alert-info }
 
+##### A.4.5 Templates
+
+To facilitate the work of selectively forwarding portions of a scalable video bitstream to each endpoint in a video conference, as is done by a Selective Forwarding Unit (SFU), for each packet, several pieces of information are required. To reduce overhead, repetitive information can be predefined and sent once. Subsequent packets refer to a template containing predefined information. In particular, when an encoder uses an unchanging (static) prediction structure to encode a scalable bitstream, parameter values used to describe the bitstream repeat in a predictable way. The techniques described in this document provide means to send repeating information as predefined templates that can be referenced at future points of the bitstream. Since a reference index to a template requires fewer bits to convey than the associated structures themselves, header overhead can be substantially reduced.
```
Comment (Philipel-WebRTC) [inline av1-rtp-spec.md:1123]: "... by a Selective Forwarding Unit (SFU) ..."

Comment (Philipel-WebRTC) [inline av1-rtp-spec.md:1123]: In the intro we call this "scalable video stream", maybe we should match that?

File: av1-rtp-spec.md (hunk 1 above)
Comment (DanilChapovalov) [reply av1-rtp-spec.md]: oops, switched to SFM everywere in the added and updated text.

File: av1-rtp-spec.md (hunk 2 above)
Comment (DanilChapovalov) [reply av1-rtp-spec.md]: applied similar changes that we did in the introduction section

Comment (DanilChapovalov) [reply av1-rtp-spec.md]: switched to SFM

**Comment by DanilChapovalov** [comment]
Updated SFU to SFM in this PR (but there still will be a merge conflict with PR#139 )

**Pull request merged**